src/main/scala/cromwell/logging/CromwellLogger.scala

```diff
@@ -17,7 +17,7 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
 
     val highlightedMessage = event.getFormattedMessage
       .replaceAll("UUID\\((.*?)\\)", TerminalUtil.highlight(2, "$1"))
-      .replaceAll("`(.*?)`", TerminalUtil.highlight(9, "$1"))
+      .replaceAll("`(.*?)`", TerminalUtil.highlight(99, "$1"))
```

Comment:
Stowaway change, this makes stuff between backticks purple instead of orange that looks too much like red on some displays.